backbone.js

```diff
@@ -613,43 +629,48 @@
 
       options = _.extend({validate: true, parse: true}, options);
       var wait = options.wait;
+      var success = options.success || _.identity;
+      var error = options.error;
+      delete options.success;
+      delete options.error;
```

Comment:
I don't want them called by ajax (i.e. jQuery or whatever it might be), because I want them to be apart of the promise chain.